exceptions.py

```diff
@@ -0,0 +1,2 @@
+class TswiftError(Exception):
+    """Base exception for errors raised in tswift"""
```

Comment:
This should not be a separate file, for a couple reasons:
1. The `setup.py` only [declares](https://github.com/brenns10/tswift/blob/master/setup.py#L15) that one module (tswift) is part of this package. So when this package gets uploaded to PyPI, the `exceptions.py` file will not be included.
2. Even if we updated the `setup.py` to include the `exceptions` module, we'd encounter another problem. `exceptions` is a common name for Python modules, and as such we would be overwriting a namespace that many people would be using. A better module to put it in would be `tswift.exceptions`, but that would require reorganizing more code.

So please put this definition into `tswift.py`

Reply:
Thanks for the explanation. I pushed an update moving the exception into the `twsift.py` file